data/data/aws/bootstrap/variables.tf

```diff
@@ -67,3 +67,8 @@ variable "vpc_security_group_ids" {
   default     = []
   description = "VPC security group IDs for the bootstrap node."
 }
+
```

Comment:
Isn't this valid for both `libvirt` and `aws` ?

Reply:
libvirt had that already.